chcemvediet/apps/inforequests/tests/test_models/test_branch.py

```diff
@@ -543,37 +540,61 @@ def test_can_add_action_method_with_multiple_arguments(self):
+            branch = Branch.create(
+                obligee=self.obligee,
+                inforequest=self.inforequest,
+                action_kwargs=dict(
+                    type=Action.TYPES.CLARIFICATION_RESPONSE,
+                    legal_date=naive_date(u'2010-10-05'),
+                ),
+            )
+
     def test_add_expiration_if_expired_method(self):
-        tests = (                                   # Expected action type,      branch, scenario
-                (Action.TYPES.REQUEST,                Action.TYPES.EXPIRATION,        0, []),
-                (Action.TYPES.CLARIFICATION_RESPONSE, Action.TYPES.EXPIRATION,        0, [u'clarification_request', u'clarification_response']),
-                (Action.TYPES.APPEAL,                 Action.TYPES.APPEAL_EXPIRATION, 0, [u'expiration', u'appeal']),
-                (Action.TYPES.CONFIRMATION,           Action.TYPES.EXPIRATION,        0, [u'confirmation']),
-                (Action.TYPES.EXTENSION,              Action.TYPES.EXPIRATION,        0, [u'extension']),
-                (Action.TYPES.ADVANCEMENT,            None,                           0, [u'advancement']),
-                (Action.TYPES.CLARIFICATION_REQUEST,  None,                           0, [u'clarification_request']),
-                (Action.TYPES.DISCLOSURE,             None,                           0, [u'disclosure']),
-                (Action.TYPES.REFUSAL,                None,                           0, [u'refusal']),
-                (Action.TYPES.AFFIRMATION,            None,                           0, [u'refusal', u'appeal', u'affirmation']),
-                (Action.TYPES.REVERSION,              None,                           0, [u'refusal', u'appeal', u'reversion']),
-                (Action.TYPES.REMANDMENT,             Action.TYPES.EXPIRATION,        0, [u'refusal', u'appeal', u'remandment']),
-                (Action.TYPES.ADVANCED_REQUEST,       Action.TYPES.EXPIRATION,        1, [u'advancement']),
-                (Action.TYPES.EXPIRATION,             None,                           0, [u'expiration']),
-                (Action.TYPES.APPEAL_EXPIRATION,      None,                           0, [u'refusal', u'appeal', u'appeal_expiration']),
+        tests = (                                   # legal date,                expected action type,      branch, scenario
+                (Action.TYPES.REQUEST,                naive_date(u'2010-07-15'), Action.TYPES.EXPIRATION,        0, []),
+                (Action.TYPES.CLARIFICATION_RESPONSE, naive_date(u'2010-07-15'), Action.TYPES.EXPIRATION,        0, [u'clarification_request', u'clarification_response']),
+                (Action.TYPES.APPEAL,                 naive_date(u'2010-07-20'), Action.TYPES.APPEAL_EXPIRATION, 0, [u'expiration', u'appeal']),
+                (Action.TYPES.CONFIRMATION,           naive_date(u'2010-07-15'), Action.TYPES.EXPIRATION,        0, [u'confirmation']),
```

Comment:
@martinmacko47 `legal_date` sa zadava od hodnoty `deadline.deadline_date`, ktore som si rucne vypocital podla kalendara. U `CONFIRMATION` to je deadline predoslej akcie, ktory sa deafultne vytvara v `create_inforequest_scenario`, V kazdom scenario sa najpr sa vytvori `REQUEST` Action. Napadlo mi, ze by mozno bolo fajn, ak to je v teste explicitne uvedene:
```
Action.TYPES.CONFIRMATION,           naive_date(u'2010-07-15'), Action.TYPES.EXPIRATION,        0, [u'request', u'confirmation']),
```
alebo aj scenario `['disclosure', 'confirmation']`, sice to nie je logicky spravne, ale ide mi o to aby som si vedel odvodit ten deadline. 
Ale v podstate to je jedno ani to tam nemusi byt.

Reply:
Ano, takto rucne vypocitane a uvedene v "tabulke" testov je fajn. Nech si to vieme kedykolkvek rucne prepocitat a porovnat v pripade potreby.

Reply:
:ok:, to iste plati aj pre `EXTENSION`